### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fgarci03**
Ran into some trouble installing, figured this could be helpful:

- fixed typo in the repository name (git clone xxx);
- added requirement for npm@3.x (npm install fails with unmet peerDependencies for webpack and eslint-plugin-react in npm@2.x);
- added instalation of py-requirements/base.txt.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,13 +63,18 @@ Remember that when you copy this repository for a new project you need to add th
...
+* `$ npm install -g npm@next` # only if version is lower than 3.x
+
 ### Main Project
 
 * `$ npm install`
 * `$ npm run dev`  # will run webpack with watch and compile code as it changes
 
 * `$ virtualenv -p /usr/bin/python3 virtualenv`
 * `$ source virtualenv/bin/activate`
+* `$ pip install -r py-requirements/base.txt`
```
Comment (almeidarruben) [inline README.md]: When you run `pip install -r py-requirements/dev.txt` you are requiring/installing the base dependencies

See first line of `py-requirements/dev.txt`: `-r base.txt`

Comment (fgarci03) [inline README.md]: Aha! Didn't notice that :)

It must have been a coincidence then, I had some problems installing the dev.txt requirements, so I tried to install base.txt before, and everything went well, so I assumed that was the issue.
Thanks for the clarification!

**Comment by lfrodrigues** [comment]
Please update PR to remove line  `pip install -r py-requirements/base.txt` and we'll merge.

**Comment by fgarci03** [comment]
Done. Thanks!

**Comment by lfrodrigues** [comment]
Please rebase to have only one commit

**Comment by fgarci03** [comment]
Sorry for the delay, first time doing a rebase so had to wait untill I got home to figure it out. I believe it's correct now. If not, please let me know

**Pull request merged**